Games/Silicon8/files.py

```diff
@@ -0,0 +1,12 @@
+def loadinto(filename, memory):
+    try:
+        file = open(filename, 'rb')
+        file.seek(0,2)
+        size = file.tell()
+        if size > len(memory):
+            return -1
+        file.seek(0)
+        file.readinto(memory)
+        return size
+    except Exception as err:
+        print('Could not read CH8 file ' + filename + ':', err)
```

Comment:
This message isn't correct, plus we will want to throw this exception and handle it elsewhere